### PR TITLE
Match organizing container

### PR DESCRIPTION
**Pull request by CarveJ**
The view which users will go to after that have clicked on the match card in the matches container to organize when and where they will meet. 

matches ---> matchDetails (organizing the match location + time)

### CODE REVIEW

File: versus-fe/src/components/OpponentDetails/OpponentDetails.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PastMatchCard from '../PastMatchCard/PastMatchCard'
+import './OpponentDetails.css'
+class OpponentDetails extends Component {
+
+  renderMatch_history = () => {
+    const previousMatches = this.props.theOpponent.match_history
+    return previousMatches.map( previousMatch => {
+      return <PastMatchCard pastMatch={previousMatch}/>
```
Comment (OliWalker) [inline versus-fe/src/components/OpponentDetails/OpponentDetails.js]: can be a one liner w/o return in map

File: versus-fe/src/App.js [hunk 2]
```diff
@@ -19,24 +21,29 @@ import Drawer from './components/Drawer/Drawer';
 class App extends Component {
   constructor(props) {
     super(props);
+    console.log('user', this.props.user.user_id)
```
Comment (arol) [inline versus-fe/src/App.js]: Remove `console.logs` always. Your DevTools console is going to be a mess after some time.

File: versus-fe/src/components/MatchDetailsInfo/MatchDetailsInfo.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    return (
+      <div className="infoBackGround">
+        <div className="infoContainer">
+
+            <form className="infoForm">
+              {this.renderInput("Location","text","location")}
+              {this.renderInput("Time","datetime-local","match_datetime")}
+            </form>
+
+            <div className="enter" onClick={this.sendFormInfo}>
```
Comment (arol) [inline versus-fe/src/components/MatchDetailsInfo/MatchDetailsInfo.js]: Always do it using the form submit action, this way the enter key will be always submitting it.

File: versus-fe/src/containers/MatchDetails/MatchDetails.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    const IdOfMatch = parseInt(IdParam, 10)
+    const matchedWithId = this.props.matches.filter( theMatch => theMatch.matches_id === IdOfMatch )
+    this.setState( {aMatch: {...matchedWithId} } )
+    this.props.getOpponent()
+  }
+
+  renderOpponentStats = () => {
+    if(this.state.opponentNow !== 'false') {
+        return <OpponentDetails theOpponent={this.state.opponentNow}/>
+      }
```
Comment (arol) [inline versus-fe/src/containers/MatchDetails/MatchDetails.js]: indentation.

**Pull request merged**